### ISSUE TITLE
Secure RTSP server

### ISSUE THREAD

**Issue by dhomas1** [openipcamera/openipc-firmware#12]
The RTSP stream is not secured. If a user would like to view the RTSP stream remotely, it would be open for anyone else to see. There is a way to secure the stream with a username and password. See here for instructions:
https://github.com/samtap/fang-hacks/issues/6#issuecomment-318861436
and here for the binary:
https://github.com/samtap/fang-hacks/issues/6#issuecomment-314940230
It would be nice if this binary (or similar) could be included with the base image. Then, it would be nice if the username/password could be defined via the web interface.

**Comment by openipcamera** [comment]
This will be in the next build.

**Comment by dhomas1** [comment]
Just a quick comment about the replacement rtsp binary from the above link (which I've installed myself). It makes the stream 720p instead of 1080p. Also, the camera is no longer recognized as a Xiaomi XiaoFang in applications like tinyCam, and requires me to set it up with the "Camera brand" set to "(Generic)" and the "Camera model" set to "Generic URL". I haven't found any way around this so far, but it may cause issues to users upgrading from previous firmwares (if the plan was to incorporate the linked binary into the OpenIPC firmware).

**Comment by newsera** [comment]
Has this been integrated?

**Comment by claell** [comment]
Yes, some update would be nice, since I think it is stated somewhere that secure streams are supported. Also the problems @dhomas1 mentioned should be considered.
